bee-api/bee-rest-api/src/endpoints/api/v1/receipts_at.rs

```diff
@@ -0,0 +1,51 @@
+use crate::{
```

Comment:
Copyright header

Reply:
Maybe do an fmt pass, it should warn about that

Reply:
Should all be addressed now, thanks!